app/views/authentications/login.html.erb

```diff
@@ -0,0 +1,14 @@
+<% set_page_title(t("page_titles.login")) %>
+
+<%= login_button(is_e2e_user: @is_e2e_user) %>
+
+<% content_for :body_end do %>
+  <script>
+    (function() {
+      var loginButton = document.querySelector('[data-module="login-button"]');
+      if (loginButton && typeof loginButton.click === 'function') {
+        loginButton.click();
+      }
+    })();
```

Comment:
The alternative to inlining this code is to either put it in the application bundle or add a new entrypoint to be loaded separately.

I could be convinced with either option. I inlined it so it will load quickly and stuck it at the bottom of the body to not block rendering.

I'm not sure whats best. I haven't provided tests for this code because it's so simple and is tested by the feature tests and e2e tests.

Reply:
I think what you've done here looks good to me